src/modules/python/pbs/v1/_pmi_cray.py

```diff
@@ -538,4 +538,28 @@ def _pmi_ramp_up(self, hosts):
                         launch(func, cmd)
                         sleep_time = random.randint(1,10)
                         time.sleep(sleep_time)
-        return True
\ No newline at end of file
+        return True
+
+    def _pmi_power_status(self, hosts):
+        pbs.logmsg(pbs.LOG_DEBUG, "Cray: status of the nodes")
+        nidset = nodenids(hosts)
+        nids, _ = nidlist(None, nidset)
```

Comment:
Instead of passing None as the first argument, can you just pass a named argument for the second?  Or is the first argument required?

Reply:
Done. Made first argument too optional and passed a named argument.